docs/contracts/errors.rst

```diff
@@ -0,0 +1,72 @@
+.. index:: ! error, revert
+
+.. _errors:
+
+*******************************
+Errors and the Revert Statement
+*******************************
```

Comment:
Won't people just figure that out?

Reply:
Some definitely will but I think that showing how we intend the feature to be used might alleviate some frustrations with things that cannot be done. I.e. instead of just telling people they cannot have duplicate error signatures it's better to show them how to use errors without running into that problem in the first place.

Anyway, just a suggestion. I don't think it's critical to the PR but it would be nice to have it.